jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/DataSetDefsBootstrap.java

```diff
@@ -35,149 +36,120 @@
 public class DataSetDefsBootstrap {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DataSetDefsBootstrap.class);
-    private static final  String JBPM_DATA_SOURCE = "${"+ KieServerConstants.CFG_PERSISTANCE_DS + "}";
+    private static final String JBPM_DATA_SOURCE = "${"+ KieServerConstants.CFG_PERSISTANCE_DS + "}";
+    private static final String SQL_SELECT_COMMON_COLS = "" + //See addBuilderCommonColumns()
+            "t.activationTime, " +
+            "t.actualOwner, " +
+            "t.createdBy, " +
+            "t.createdOn, " +
+            "t.deploymentId, " +
+            "t.description, " +
+            "t.dueDate, " +
+            "t.name, " +
+            "t.parentId, " +
+            "t.priority, " +
+            "t.processId, " +
+            "t.processInstanceId, " +
+            "t.processSessionId, " +
+            "t.status, " +
+            "t.taskId, " +
+            "t.workItemId, " +
+            "pil.correlationKey, " +
+            "pil.processInstanceDescription, " +
+            "lmd.lastModificationDate ";
+    
+    private static final String SQL_SUBQUERY_LAST_MOD_DATE = "(" +
+            "select te.taskId, max(te.logTime) as lastModificationDate " +
```

Comment:
fully agree, that will be much better approach and not so big issue as we can provide update script to populate that column with data based on TaskEvent anyway